docs/the-guide/chapter-2-share-your-ml-experiment-code-with-git/index.md

```diff
@@ -95,7 +95,10 @@ Additionally, this will help to ensure that the repository size and clone time
 remain optimized.
 
 
-```sh title="Execute the following command(s) in a terminal"
+```sh title=".gitignore"
+## Macos Specific
+.DS_Store
+
```

Comment:
I would add a note such as "If using macOS, you might want to ignore `.DS_Store` files as well to avoid pushing Apple's metadata files to your repository." and remove the rest of the mentions in order not to pollute the reading for this specific case. What do you think?